app/email/email.go

```diff
@@ -567,6 +567,7 @@ func (es *Service) SendGuestInviteEmails(
 					"channels": strings.Join(channelIDs, " "),
 					"email":    invite,
 					"guest":    "true",
+					"senderId": senderUserId,
```

Comment:
when sending the invitation to the guest we needed to store the senderId along with the token information in the DB, so when signing up the validation process could use it to validate the permissions for the sender to make sure he still has permissions on the team/channels he is inviting.